src/config/events.ts

```diff
@@ -3198,6 +3341,30 @@ const allEvents = [
     Youtube: 'https://www.youtube.com/channel/UCcTmSSIs_rRyUOlQKJZ4lpg',
     '📮': ''
   },
+  {
```

Comment:
Why this empty event?

Reply:
this is how we are getting it from the notion page: https://www.notion.so/Database-of-blockchain-conferences-63a2b13af8a647c39231f17f7756f4f6

This should not appear in the UI though

Reply:
Got it!